### PR TITLE
Update cli.test_computeresource

### PR DESCRIPTION
**Pull request by elyezer**
Adjust the invalid and valid data generators to match the new behavior.

Test results:

```
$ py.test tests/foreman/cli/test_computeresource.py -n auto --boxed       
========================== test session starts ===========================
platform linux -- Python 3.4.2, pytest-2.9.1, py-1.4.31, pluggy-0.3.1
rootdir: /home/elyezer/code/robottelo, inifile: 
plugins: cov-2.2.1, testmon-0.8.2, xdist-1.14
gw0 [14] / gw1 [14] / gw2 [14] / gw3 [14]
scheduling tests via LoadScheduling
s.............
================= 13 passed, 1 skipped in 228.97 seconds =================
```

### CODE REVIEW

File: tests/foreman/cli/test_computeresource.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def test_positive_create_with_name(self):
 
         """
         ComputeResource.create({
-            'name': gen_string(str_type='alpha'),
+            'name': 'cr {0}'.format(gen_string(str_type='alpha')),
```
Comment (sthirugn) [inline tests/foreman/cli/test_computeresource.py]: is there a need for this change?

Comment (elyezer) [inline tests/foreman/cli/test_computeresource.py]: That is to ensure that a Compute Resource can also have spaces when being created. In this way we get coverage for the new behavior for both create and update.

Comment (sthirugn) [inline tests/foreman/cli/test_computeresource.py]: :+1:

**Comment by sthirugn** [comment]
ACK and merge

**Pull request merged**